### PR TITLE
Update translation

### PR DESCRIPTION
**Pull request by fisker**
- Improved translation for Simplified Chinese
- Add missing full stop sign. Fix inconsistent full stop sign in other languages.
- Add an example to `ext.config.prettierPath` from other languages, find it's useful. (Not sure if it works for relative path)
- Add options list for `quoteProps` (from prettier.io)
- Remove `If true, ` in these places, it's a checkbox before it, doesn't make sense to set to `true`. (We can use `If enabled`, if you feel necessary)

### CODE REVIEW

**Comment by fisker** [comment]
@sxzz @Mrbunker Can you help to review? Since you are recently active during #2775 谢谢

File: package.nls.zh-cn.json [hunk 1]
```diff
@@ -1,35 +1,35 @@
...
-  "ext.config.singleAttributePerLine": "如果 HTML元素（包括 JSX 等）具有多个属性，将其每个属性格式化为单独占一行",
-  "ext.config.bracketSameLine": "如果 HTML元素（包括 JSX 等） 具有多个属性，将其结束标签右尖括号 `>` 放在最后一个属性的末尾，而不是另起一行",
+  "ext.command.createConfigFile.title": "Prettier：选择配置文件",
+  "ext.command.forceFormatDocument.title": "格式化文件 （强制）",
+  "ext.config.arrowParens": "箭头函数仅有一个参数时，参数是否添加括号",
+  "ext.config.bracketSpacing": "在对象字面量的花括号内侧使用空格作为间隔",
+  "ext.config.configPath": "指定 Prettier 配置文件的路径",
+  "ext.config.documentSelectors": "指定一个 [glob patterns](https://code.visualstudio.com/api/references/vscode-api#GlobPattern) 列表，使匹配的文件也启用 Prettier",
+  "ext.config.endOfLine": "指定 Prettier 使用的换行符。",
+  "ext.config.htmlWhitespaceSensitivity": "指定 HTML 文件的空白字符敏感度。\n可选项：\n- `css` - 以 CSS 的 display 属性为准。\n- `strict` - 空白字符敏感。\n- `ignore` - 空白字符不敏感。",
```
Comment (fisker) [inline package.nls.zh-cn.json:9]: 不太确定这个怎么表达, 但还是应该参考英文版.

File: package.nls.zh-cn.json [hunk 2]
```diff
@@ -1,35 +1,35 @@
...
-  "ext.config.useTabs": "使用 `Tab` 缩进，而不是空格缩进",
+  "ext.config.trailingComma": "指定添加尾后逗号的方式。\n选项：\n- `none` - 无尾后逗号。\n- `es5` - 在 ES5 中有效的尾后逗号（如对象与数组等）。\n- `all` - 尽可能添加尾后逗号（如函数参数）。",
+  "ext.config.useEditorConfig": "使用 `.editorconfig` 中的配置选项。详见 [`prettier.resolveConfig`](https://prettier.io/docs/en/api.html) 文档。",
+  "ext.config.useTabs": "使用制表符缩进，而不是空格缩进。",
   "ext.config.vueIndentScriptAndStyle": "缩进 Vue 的 `<script>` 和 `<style>` 标签。",
-  "ext.config.embeddedLanguageFormatting": "对引号中代码使用智能格式化",
-  "ext.config.enable": "启用 Prettier。更改后需要重启 VS Code",
-  "ext.config.enableDebugLogs": "启用调试日志",
-  "ext.capabilities.untrustedWorkspaces.description": "在不受信任模式下只有内置版本的 Prettier 会生效"
+  "ext.config.embeddedLanguageFormatting": "控制嵌入的其他语言代码格式化",
```
Comment (fisker) [inline package.nls.zh-cn.json:31]: 这里也是

File: package.nls.zh-cn.json (hunk 1 above)
Comment (Mrbunker) [reply package.nls.zh-cn.json:9]: 「指定 HTML 文件中空格字符的格式化方式」或者「指定 HTML 文件中空格字符的格式化敏感度」

**敏感度** 这个词肯定要额外去理解才能懂，光看描述应该很难明白。
可以考虑贴参考链接 [whitespace-sensitive formatting](https://prettier.io/blog/2018/11/07/1.15.0.html#whitespace-sensitive-formatting)

File: package.nls.zh-cn.json [hunk 3]
```diff
@@ -1,35 +1,35 @@
...
-  "ext.config.ignorePath": "指定 `.prettierignore` （或其他配置方案文件）的路径。[参考文档](https://prettier.io/docs/en/ignore.html#ignoring-files-prettierignore) ",
-  "ext.config.insertPragma": "格式化的同时自动插入 `@format` 的特殊注释，以表示该文件已被 Prettier 格式化过",
-  "ext.config.singleAttributePerLine": "如果 HTML元素（包括 JSX 等）具有多个属性，将其每个属性格式化为单独占一行",
-  "ext.config.bracketSameLine": "如果 HTML元素（包括 JSX 等） 具有多个属性，将其结束标签右尖括号 `>` 放在最后一个属性的末尾，而不是另起一行",
+  "ext.command.createConfigFile.title": "Prettier：选择配置文件",
+  "ext.command.forceFormatDocument.title": "格式化文件 （强制）",
+  "ext.config.arrowParens": "箭头函数仅有一个参数时，参数是否添加括号",
+  "ext.config.bracketSpacing": "在对象字面量的花括号内侧使用空格作为间隔",
+  "ext.config.configPath": "指定 Prettier 配置文件的路径",
+  "ext.config.documentSelectors": "指定一个 [glob patterns](https://code.visualstudio.com/api/references/vscode-api#GlobPattern) 列表，使匹配的文件也启用 Prettier",
```
Comment (Mrbunker) [inline package.nls.zh-cn.json:7]: 这里有些地方句号没补齐

Comment (fisker) [reply package.nls.zh-cn.json:7]: 这里除了 `bracketSpacing`, 其他我故意没加 因为后面是输入框或者下拉框.

Comment (fisker) [reply package.nls.zh-cn.json:7]: [Add full stop to bracketSpacing option](https://github.com/prettier/prettier-vscode/pull/2860/commits/814d8c8b124f191617d86a48df72f6c8d72412aa)

Comment (fisker) [reply package.nls.zh-cn.json:7]: [Add link to "whitespace sensitivity" in all languages](https://github.com/prettier/prettier-vscode/pull/2860/commits/812e0b22a26fd49a1b2c31e2c0315a1a694405ed)

Comment (Mrbunker) [reply package.nls.zh-cn.json:7]: > 这里除了 `bracketSpacing`, 其他我故意没加 因为后面是输入框或者下拉框. 主要是因为英文的没加. 你觉得需要吗?

啊，我是没注意这么细节的地方，我是认为可加可不加啦

**Comment by ntotten** [comment]
Should we get this merged?

**Comment by fisker** [comment]
> Should we get this merged?

Ready.

---

@Mrbunker Thank you for the help.

**Pull request merged**